### ISSUE TITLE
Display SOR and/or FS

### ISSUE THREAD

**Issue by dnil** [Clinical-Genomics/scout#2228]
StrandBias used to be an important indicator of allele call quality. With the advent of wgs, it seems to have lost popularity, but I got a question from a customer about it yesterday. GATK still calculates a few measures related to it (StrandOddsRatio INFO.SOR and FisherStrand INFO.FS). Perhaps show these on the genotype quality panel? This will likely change anyway with DV, but I assume we will still want to support GATK for some time to come.

**Comment by vwirta** [comment]
This could be very valuable for cancer panel (and exome) analyses as well, but not sure what these fields are called in the various somatic callers (if they are calculated at all)

**Comment by dnil** [comment]
Indeed! Some most likely do: eg FreeBayes and samtools had similar-but-different ones. We can inventory the ones currently loaded from Balsamic and check.
